plugins/Denizen/scripts/cooking/allergies.dsc

```diff
@@ -0,0 +1,77 @@
+allergies:
+    type: world
+    debug: false
+    events:
+        on player dies:
+            - if <util.random_chance[12]>:
+                - define allergy <script[allergy_info].data_key[valid_allergies].keys.random>
+                - define effect <script[allergy_info].data_key[valid_effects].keys.random>
+                - flag player allergic:<[allergy]>
+                - flag player allergy_effect:<[effect]>
+                - stop
+            - flag player allergic:!
+            - flag player allergy_effect:!
+
+        on player consumes item flagged:allergic:
+
+            - define allergies <script[allergy_info].data_key[valid_allergies.<player.flag[allergic]>]>
+            - if <[allergies].contains[<context.item.script.name.if_null[<context.item.material.name>]>]>:
+                - foreach <script[allergy_info].data_key[valid_effects.<player.flag[allergy_effect]>]> as:a key:b:
+                    - cast <[b]> duration:<[a].get[duration]> amplifier:<[a].get[amplifier]> hide_particles no_icon
+
+allergy_info:
+    type: data
+    valid_allergies:
+        seafood:
+            - raw_salmon
+            - cooked_salmon
+            - raw_cod
+            - cooked_cod
+            - pufferfish
+            - dried_kelp
+        wheat:
+            - bread
+            - baguette
+            - cake
+            - cookie
+        diary:
+            - milk_bucket
+            - cake
+        pumpkin:
+            - pumpkin_pie
+            - pumpkin
```

Comment:
```suggestion

```
You can't eat pumpkins.

Reply:
just because of this i will make it so you can